pytorch_lightning/trainer/trainer.py

```diff
@@ -483,12 +484,23 @@ def __init__(
                     profiler = AdvancedProfiler()
                     trainer = Trainer(profiler=profiler)
 
+            enable_benchmarking (bool): If true enables benchmarking.
+                Example::
+
+                    # default disabled by the Trainer
```

Comment:
I do not think this need example like this...

Reply:
haha i followed the style of previous lines. let me remove this

Reply:
wait a sec, the other options are enabled by default right? and this option i think should be disabled by default

Reply:
my point was to write a sentence what the default value is, do not pass redundant example...

Reply:
well I think `default disabled by the Trainer` is explicit enough.. I made a little change. please take a look :smile:

Reply:
the style was pruned recently and all examples saying this is default value was removed... ^^